src/rocm_docs/core.py

```diff
@@ -9,8 +9,11 @@
 from abc import ABC, abstractmethod
 from pathlib import Path
 from typing import Any, Callable, Dict, Generic, List, Type, TypeVar
-from bs4 import BeautifulSoup
+import importlib_resources
+import datetime
+import time
 
+import bs4
```

Comment:
NIT: Could you please run isort to sort these imports?

Reply:
Done in [a84f63f](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/139/commits/a84f63fcfe0e77d60591a27d2e882c3bd812e212)